scvi/data/_utils.py

```diff
@@ -11,6 +11,12 @@
 import scipy.sparse as sp_sparse
 from anndata import AnnData
 from anndata._core.sparse_dataset import SparseDataset
+
+try:
```

Comment:
can you add a todo here to use the experimental api once we lower bound to anndata 0.8?